digitalpy/core/zmanager/action_mapper.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC
+from digitalpy.core.zmanager.request import Request
+
+from digitalpy.core.zmanager.response import Response
+
+
+class ActionMapper(ABC):
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D101 Missing docstring in public class